package.json

```diff
@@ -13,8 +13,8 @@
     "startWeb": "nodemon server/index.js",
     "openPage": "open http://localhost:3003",
     "test": "jest",
-    "loadData": "sed -i '' '16,$ d' database-postgresql/schema.sql",
-    "seedData": "npm run loadData; node database-postgresql/seed.js; psql trailPhotosDB < database-postgresql/schema.sql"
+    "loadData": "psql trailPhotosDB < database-postgresql/schema.sql",
+    "seedData": "sed -i '' '16,$ d'; node database-postgresql/seed.js; npm run loadData"
```

Comment:
per discussion on start script `"start": "nodemon server/index.js | npm run react-dev | npm run openPage"`

Reply:
@ryanbrennan12, this should be fixed - please repull and lmk - thanks!

Reply:
all good